detroit-ui-components/src/page_components/listing/ListingCard.tsx

```diff
@@ -41,7 +39,7 @@ export interface ListingCardProps {
   footerContent?: React.ReactNode
   imageCardProps: ImageCardProps
   stackedTable?: boolean
-  tableProps?: ListingCardTableProps
+  tableProps?: any
```

Comment:
Temp fix for everything to play nicely, fixed soon in the individual component sync tickets